src/fides/api/models/privacy_experience.py

```diff
@@ -28,32 +28,32 @@ class ComponentType(Enum):
     privacy_center = "privacy_center"
 
 
-class DeliveryMechanism(Enum):
+class BannerEnabled(Enum):
     """
-    The delivery mechanism - not formalized in the db
+    Whether the banner should display
     """
 
-    banner = "banner"
-    link = "link"
+    always_enabled = "always_enabled"
+    enabled_where_required = "enabled_where_required"  # If the user's region has at least one opt-in or notice-only notice
+    always_disabled = "always_disabled"
 
 
 class ExperienceConfigBase:
     """Base schema to share common experience config."""
 
-    acknowledgement_button_label = Column(String)
```

Comment:
Changing to `acknowledge_button_label`

Reply:
nice, that lets us not have to suffer from the question `acknowledgment` or `acknowledgement` 😄

Reply:
hahaha